reactor-core/src/test/java/reactor/core/publisher/FluxSwitchMapTest.java

```diff
@@ -16,21 +16,66 @@
 
 package reactor.core.publisher;
 
-import java.util.concurrent.ConcurrentLinkedQueue;
+import java.time.Duration;
+import java.util.concurrent.RejectedExecutionException;
+import java.util.function.Consumer;
+import java.util.function.Predicate;
 
 import org.junit.jupiter.api.Test;
-import org.reactivestreams.Subscription;
-
-import reactor.core.CoreSubscriber;
+import org.junit.jupiter.api.Timeout;
+import reactor.core.Exceptions;
 import reactor.core.Scannable;
+import reactor.core.scheduler.Scheduler;
+import reactor.core.scheduler.Schedulers;
 import reactor.test.StepVerifier;
 import reactor.test.subscriber.AssertSubscriber;
-import reactor.util.concurrent.Queues;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static reactor.core.publisher.Sinks.EmitFailureHandler.FAIL_FAST;
 
 public class FluxSwitchMapTest {
+	@Test
+	@Timeout(120_000)
+	public void test() {
+		final Scheduler scheduler = Schedulers.newSingle("test");
+		final Scheduler scheduler2 = Schedulers.newParallel("test2");
+		Flux<Integer> integerFlux = Flux.defer(() -> Flux.range(0, 20)
```

Comment:
consider not using short lambda here to reduce the indentation.


`() -> something`
vs
```
() -> {
    return something;
}
```

Reply:
done